ISISNeutronMuon/MDANSE
# Fix PDF totals having zero values

**Description of work.**

Fixes #56 

Adding RDF and TCF calculations in #40 introduced a bug where the total were calculated incorrectly. This was because the calculation of pdf_total for atom pairs was forgotten to be modified, and so it used the `self._outputData` object which was empty. This has been fixed by replacing the empty references with the appropriate, calculated arrays.

**To test:**

1. Run PDF analysis using the included trajectory waterbox_in_periodic_universe.nc
2. Observe that `pdf_total_OO` and `pdf_total` have non-zero values.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**.
